opds.py

```diff
@@ -219,7 +288,7 @@ def categories(cls, work):
         return categories
 
     @classmethod
-    def authors(cls, work, license_pool, edition, identifier):
+    def authors(cls, work, edition):
```

Comment:
Several methods have had LicensePool and Identifier removed from their signatures to guarantee they don't have access to that information.